apps/explorer/components/Address/index.tsx

```diff
@@ -279,45 +308,55 @@ function formatPayoutEntity(payout: ExplorerEvent): EntityListItemProps {
     height: payout.index.height,
     sc: new BigNumber(siacoinElement.siacoinOutput.value),
     timestamp: new Date(payout.timestamp).getTime(),
+    maturityData: {
+      currentHeight,
+      maturityHeight: siacoinElement.maturityHeight,
+    },
   }
 }
 
 function formatUnspentSiacoinOutputEntity(
-  siacoinOutput: ExplorerSiacoinOutput
+  siacoinOutput: ExplorerSiacoinOutput,
+  currentHeight: number
 ): EntityListItemProps {
   return {
     hash: siacoinOutput.id,
     sc: new BigNumber(siacoinOutput.siacoinOutput.value),
     label: 'siacoin output',
     initials: 'SO',
     scVariant: 'value' as const,
+    maturityData: {
```

Comment:
Two little tweaks:
- we should add an `isMature` here that is `const isMature = e.maturityHeight <= nodeBlockHeight` and use that to render locked or unlocked.
- For the bottom height in the UI, instead of current network height we should pass the height of the specific event
That way the bottom number is the height when the txn event occurred, the top number is when it unlocks, and the lock vs unlock is determined by the network height.